Packet++/header/NtpLayer.h

```diff
@@ -0,0 +1,615 @@
+#ifndef PACKETPP_NTP_LAYER
+#define PACKETPP_NTP_LAYER
+
+#include "Logger.h"
+#include "IPv4Layer.h"
+#include "Layer.h"
+
+#include "GeneralUtils.h"
+#include "SystemUtils.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+    /**
+	 * @class NtpLayer
+	 * Represents a NTP (Network Time Protocol) layer
+     * 
+     * @brief The NTP packet consists of an integral number of 32-bit (4 octet) words in network byte order. 
+     * The packet format consists of three components: the header itself, one or more optional extension fields (for v4), 
+     * and an optional message authentication code (MAC). Currently the extension fields are not supported. The NTP header is:
+     * 
+     * @verbatim 
```

Comment:
nit: this asterisk (`*`) is redundant:
![image](https://user-images.githubusercontent.com/9059541/151922333-65509eb8-8260-4281-9bb7-7a7bd04569de.png)